### DISCUSSION TITLE
パッケージ化後のソフトが開きません(ver0.11.1)

### DISCUSSION THREAD

**Discussion by mikuro898** [HANON-games/Shiden#25] [Q&A] [answered]
いつもお世話になっています。

先日、Ver.0.11.1のShidenプラグインを導入しました。
ゲームの動作確認のためにパッケージ化をしたのですが、パッケージ化した後のソフト(.exe)が開けませんでした。試しに初期状態のプロジェクトにShidenプラグインのみを入れてVer5.5、5.4、5.3で同じようにパッケージ化したのですが、そのすべてでソフト(.exe)が開けませんでした。パッケージ化自体はエラーなく正常に完了します。

私の環境によるものかなとも思ったのですが、Ver0.10.9バージョンまでは正常に機能しているのでプラグインの方に問題がある気がしています。windowソフトのパッケージ化です。

お忙しいところ恐れ入りますが、ご確認いただけますと幸いです。
どうぞよろしくお願いいたします。

**Comment by HANON-games** [comment, marked as answer]
@mikuro898 
Shiden をご利用いただきありがとうございます！
返信が遅くなってしまい申し訳ありません。

確認したところ Ver.0.11.0 における変更により、パッケージ化後に正常にウィジェットが開けない問題を確認しました。
近日中に修正予定ですので、ご不便おかけし申し訳ありませんが暫くお待ちいただけますと幸いです。

なお、今後は以下の issue にて管理できればと思います。
https://github.com/HANON-games/Shiden/issues/26
